Sdk/Speckle.Connectors.Utils/Operations/AccountService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Speckle.Core.Credentials;
+
+namespace Speckle.Connectors.Utils.Operations;
+
+public class AccountService
```

Comment:
please add comment on how this class is expected to be injected around, to prevent fuckups like we had in the past when we changed registration and everything went haywire!

Reply:
you are good damn right. doing it